research-app/src/App.vue

```diff
@@ -52,10 +67,37 @@
                   label="name"
                   placeholder="Background"
                   >
+                  <template #option="option">
+                    <h4 style="margin:0">{{option.name}}</h4>
+                    <em style="margin:0; font-size:small;">{{option.description}}</em>
```

Comment:
FWIW, I believe that it's generally discouraged to include styling via the `style=` HTML attribute, even though it can get annoying to create whole CSS classes for small fixups such as these. See e.g. [the MDN docs][mdn] on `style`.

[mdn]: https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/style